### DISCUSSION TITLE
Edit config.yml file for HTTPS

### DISCUSSION THREAD

**Discussion by ctrasti** [requarks/wiki#4496] [Help / Questions]
Hi all!

I am running Wiki.JS on Debian11 and Docker. I am fairly new to docker, but I've come to understand that it's a sort of virtualized enviroment running on my Debian VM.

I've looked at the official documentation, and I am having trouble locating the config.yml file. I thought that I'd maybe find it in _**(what I thought was the)**_ the docker root folder. But there I just found these strange folders with long cryptic names.

root@xxxxxx:/var/lib/docker/containers# ls -lsa
4 drwx-----x  4 root root 4096 Sep 17 13:32 60ec99998bf89c3517ec9fa4c36529f727178e41032c885ad96f2xxxxxxxxxxxx
4 drwx-----x  4 root root 4096 Sep 17 13:32 cdfea5c2beacd6a1229024e98f96763458d1932ecdd4569f8d098xxxxxxxxxxx
4 drwx-----x  4 root root 4096 Sep 17 13:32 fc8a0da23ec242b283ed276d895651ace68fb0ddf1fca60fc7ea4xxxxxxxxxxxx

Any help would be greatly appreciated. Thanks, and have a great day.

**Comment by Phil-hostwiki** [comment]
To access the docker container, run ` docker exec -ti container_id bash`   .  
You will find the `config.yml` file there. Once done with editing, exit the container with `exit`   .  
To effect changes, restart the container with ` docker restart container_id`  .  

PS: you can also use the container_name in place of container_id.

**Comment by ieaster1** [comment]
Some additional details could be useful to understand your situation but containers should be considered as immutable.  Any changes you make within them will not survive a restart.

You would either want to use the environment variables or mount the config.yml file to the host machine.

See: https://docs.requarks.io/install/docker#alternative-mount-the-config-file
